mappings/net/minecraft/client/multiplayer/report/ChatAbuseReport.mapping

```diff
@@ -16,9 +17,9 @@ CLASS net/minecraft/unmapped/C_nfpruzew net/minecraft/client/multiplayer/report/
 			ARG 2 messageReported
 		METHOD m_ldvdzbnk getSelectedMessages ()Lit/unimi/dsi/fastutil/ints/IntSet;
 		METHOD m_pvvmpdsi toggleMessageSelected (I)V
-			ARG 1 messageId
+			ARG 1 index
 		METHOD m_rafbfmcr isMessageSelected (I)Z
-			ARG 1 messageId
+			ARG 1 index
```

Comment:
Oops I changed this to match above but must not have saved in Enigma before committing.